spec/kibana_spec.rb

```diff
@@ -1,24 +1,9 @@
 # encoding: utf-8
 require 'spec_helper'
 
-kibana_version = '4.4.1'
-
-describe file("/opt/kibana-#{kibana_version}-linux-x64.tar.gz") do
-  it { should exist }
-  it { should be_file }
-end
-
-describe file("/opt/kibana-#{kibana_version}-linux-x64") do
-  it { should exist }
-  it { should be_directory }
-  its('owner') { should eq 'www-data' }
-  its('group') { should eq 'www-data' }
-end
-
 describe file('/opt/kibana') do
   it { should exist }
-  it { should be_symlink }
-  it { should be_linked_to "/opt/kibana-#{kibana_version}-linux-x64" }
+  it { should be_directory }
```

Comment:
@conorsch Great point and one I intended to address. When you blow away the previous tarball installation (I think that's best, otherwise you get a mix of permissions and ownership and possibly outdated/unused source files laying around) and use the deb packages then everything is root-owned. With that approach, I'm seeing some failures with the systemd service now, probably because `User=www-data` is in the service definition. Working it out now...